### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tarnenaky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# finance-manager-bot
-Telegram finance manager bot that helps you to track your expenses.
+# looks-like-gauno-bot
```
Comment (aliaksanderkazlou) [inline README.md]: It's wrong statement.

**Review by aliaksanderkazlou — CHANGES_REQUESTED** [review]
Need fixes

**Comment by tarnenaky** [comment]
I think now is better

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-# looks-like-gauno-bot
+# looks-like-pedar-gauno-bot
```
Comment (aliaksanderkazlou) [inline README.md]: -

**Review by aliaksanderkazlou — CHANGES_REQUESTED** [review]
sosi

**Comment by tarnenaky** [comment]
Accept this request, pedar

**Comment by aliaksanderkazlou** [comment]
izveni, no ne

**Pull request closed without merging**